src/data/git-quiz.ts

```diff
@@ -542,13 +542,13 @@ const gitQuiz = [
     Distractor2: "Nano",
     Distractor3: "VS Code",
     Explanation:
-      "Vim stands for Vi Improved and it is terminal based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
+      "Vim stands for Vi Improved and is a terminal-based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
```

Comment:
this change is extraneous for this PR, please remove it

Reply:
Removed